include/exadg/grid/moving_mesh_base.h

```diff
@@ -41,11 +41,16 @@ class MovingMeshBase : public MappingDoFVector<dim, Number>
   /**
    * Constructor.
    */
-  MovingMeshBase(std::shared_ptr<Mapping<dim>> mapping,
-                 unsigned int const            mapping_degree_q_cache,
-                 Triangulation<dim> const &    triangulation)
-    : MappingDoFVector<dim, Number>(mapping, mapping_degree_q_cache, triangulation)
+  MovingMeshBase(std::shared_ptr<Mapping<dim> const> mapping_undeformed,
+                 unsigned int const                  mapping_degree_q_cache)
+    : MappingDoFVector<dim, Number>(mapping_degree_q_cache), mapping_undeformed(mapping_undeformed)
   {
+    // Make sure that MappingQCache is initialized correctly. An empty dof-vector is used and,
+    // hence, no displacements are added to the reference configuration described by
+    // mapping_undeformed.
+    DoFHandler<dim> dof_handler;
```

Comment:
I think this change (before we had `dof_handler(triangulation)`) introduced a bug because I get now in debug mode the error message "This DoFHandler object has not been associated with a triangulation." in the `MappingDoFVector::initialize()` function.